test_app/uwsgi.ini

```diff
@@ -7,7 +7,7 @@ chmod-socket = 660
 vacuum = true
 
 # Log to stdout
-logto = /dev/stdout
+# logto = /dev/stdout
```

Comment:
This seemed to cause the problem related to seeking, that it was logging to a file that didn't allow seeking (which was /dev/stdout). It still logged without this, and I think we weren't really getting uwsgi logs without it, maybe.